R/get_AUC.R

```diff
@@ -0,0 +1,66 @@
+#' Calculate AUC
+#'
+#' Function that calculates AUC for dose-response curves
+#'
+#' This function takes in a type of model and a set of model parameters,
+#' and return the Area under the curve (AUC) between the lower and upper
+#' concentration bounds. AUC can be used to compute an efficacy/ potency
+#' metric for "active" does-response curves. The model parameters should be
+#' entered as a list or a vector. For models operate in log scale (hill and
+#' gain-loss), the lower and upper concentration bounds, "ga" (gain AC50)
+#' and "la" (loss AC50) will be converted to log-scale if applicable.
+#' This version hasn't taken consideration into how to appropriately work with
+#' decreasing curves. One of the future works to do.
```

Comment:
In the vignette you have an example of a decreasing function and one that is below and above the x-axis (y= 0).  So not sure you need the last sentence.